go.mod

```diff
@@ -3,6 +3,11 @@ module github.com/googleinterns/ocsp-response-linter
 go 1.15
 
 require (
+	github.com/golang/mock v1.4.4
 	github.com/grantae/certinfo v0.0.0-20170412194111-59d56a35515b
 	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
+	golang.org/x/tools v0.0.0-20200828161849-5deb26317202 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 )
+
+replace github.com/googleinterns/ocsp-response-linter => /Users/granthugh/Desktop/ocsp-response-linter
```

Comment:
this line doesn't look like it should be here maybe.

Reply:
You are completely right, thank you for catching that!